docs/resources/Guild.md

```diff
@@ -722,7 +722,7 @@ Adds a user to the guild, provided you have a valid oauth2 access token for the
 
 ## Modify Guild Member % PATCH /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/members/{user.id#DOCS_RESOURCES_USER/user-object}
 
-Modify attributes of a [guild member](#DOCS_RESOURCES_GUILD/guild-member-object). Returns a 204 empty response on success. Fires a [Guild Member Update](#DOCS_TOPICS_GATEWAY/guild-member-update) Gateway event. If the `channel_id` is set to null, this will force the target user to be disconnected from voice.
+Modify attributes of a [guild member](#DOCS_RESOURCES_GUILD/guild-member-object). Returns a 200 Ok with the [guild member](#DOCS_RESOURCES_GUILD/guild-member-object) as the body on v8, or 204 No Content on v6. Fires a [Guild Member Update](#DOCS_TOPICS_GATEWAY/guild-member-update) Gateway event. If the `channel_id` is set to null, this will force the target user to be disconnected from voice.
```

Comment:
```suggestion
Modify attributes of a [guild member](#DOCS_RESOURCES_GUILD/guild-member-object). Returns a [guild member](#DOCS_RESOURCES_GUILD/guild-member-object) object. Fires a [Guild Member Update](#DOCS_TOPICS_GATEWAY/guild-member-update) Gateway event. If the `channel_id` is set to null, this will force the target user to be disconnected from voice.
```

consistency; v6 behavior is generally not documented

Reply:
Oh, will fix :+1: